srcpkgs/ol/template

```diff
@@ -9,7 +9,7 @@ maintainer="rc-05 <rc23@email.it>"
 license="LGPL-3.0-or-later, MIT"
 homepage="https://yuriy-chumak.github.io/ol/"
```

Comment:
```suggestion
homepage="https://yuriy-chumak.github.io/ol/"
changelog="https://raw.githubusercontent.com/yuriy-chumak/ol/master/doc/CHANGELOG.md"
```

Reply:
I've added and squashed into one commit.